src/auth/signup/components/signUp.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import signUpUser from '../redux/actions/signUpAction';
+import validation from '../validation';
+import '../CSS/signup.scss';
+
+const Loader = require('react-loader');
```

Comment:
@Emmanuelaaron what happens if we instead import like so `import Loader from 'react-loader` ?

Reply:
According to the `react-loader` [resource](https://www.npmjs.com/package/react-loader), you can only declare a variable of `Loader` to a `require` class with react `react-loader as an attribute